### PR TITLE
clean up TODO comments in sharedTree.bench.ts

### PR DESCRIPTION
**Pull request by daesunp**
## Description

This PR includes tests for higher node count that we couldn't do before, as we were running into a "BatchTooLarge" error. It also adds a step to update the tree schemas so that chunk compression can be used.

### CODE REVIEW

File: packages/dds/tree/src/test/scalableTestTrees.ts [hunk 1]
```diff
@@ -82,8 +82,23 @@ type JSWideTreeRoot2 = InsertableFlexField<typeof wideSchema.rootFieldSchema>;
 	type _check2 = requireAssignableTo<JSWideTree, JSWideTreeRoot2>;
 }
 
+/**
+ *
```
Comment (CraigMacomber) [inline packages/dds/tree/src/test/scalableTestTrees.ts:86]:
```suggestion
```
No need for a blank line

File: packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts [hunk 2]
```diff
@@ -45,29 +46,32 @@ import {
 	checkoutWithContent,
 	flexTreeViewWithContent,
 	insert,
+	jsonSequenceRootSchema,
 	toJsonableTree,
 } from "../utils.js";
 
-// number of nodes in test for wide trees
+// Number of nodes in test for wide trees.
```
Comment (CraigMacomber) [inline packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts:53]: I don't think it really matters, but this grammatically is a noun-phrase not a sentence (which I think is fine, as its documenting a value). Thus adding a period isn't a clear improvement. Its fine either way in my opinion.

File: packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts [hunk 3]
```diff
@@ -45,29 +46,32 @@ import {
...
 	[100, BenchmarkType.Perspective],
-	[500, BenchmarkType.Measurement],
+	[1000, BenchmarkType.Measurement],
+	[10000, BenchmarkType.Measurement],
 ];
-// number of nodes in test for deep trees
+// Number of nodes in test for deep trees.
+// TODO: We currently run into a "maximum call stack size exceeded" error if we increase the node count,
+// due to recursive calls when calling flexTreeViewWithContent, or calling assert.deepEqual.
+// To include tests with higher node counts, we will likely need to replace these parts of the code to be done iteratively.
```
Comment (CraigMacomber) [inline packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts:63]: This is not so much an issue with these tests, but a major limitation of a large portion of our tree processing code. The encoders, and decoders, editing code and more will all fail with really deep tress.

Comment (daesunp) [reply packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts]: Updated comment mentioning the current limitations of a large portion of our tree processing code.

File: packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts (hunk 2 above)
Comment (daesunp) [reply packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts]: Removed unnecessary period.

File: packages/dds/tree/src/test/scalableTestTrees.ts (hunk 1 above)
Comment (daesunp) [reply packages/dds/tree/src/test/scalableTestTrees.ts]: Removed blank line.

File: packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts [hunk 4]
```diff
@@ -389,6 +392,9 @@ describe("SharedTree benchmarks", () => {
 
 						// Setup
 						const provider = new TestTreeProviderLite(nbPeers, factory);
+						provider.trees.map((tree): void => {
+							tree.checkout.updateSchema(intoStoredSchema(jsonSequenceRootSchema));
+						});
```
Comment (CraigMacomber) [inline packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts:397]: you don't need all trees to concurrently edit the schema.
```suggestion
						provider.trees[0].checkout.updateSchema(intoStoredSchema(jsonSequenceRootSchema));
						provider.processMessages();
```

**Pull request closed without merging**